### PR TITLE
Added the megalodon pet

### PR DESCRIPTION
**Pull request by felix642**
Added the pet description to the website. It's only missing a texture.

### CODE REVIEW

**Comment by felix642** [comment]
image

File: src/constants/petStats.js [hunk 1]
```diff
@@ -1953,6 +1953,45 @@ class Squid extends Pet {
...
+		return {
+			name: "§6Enhanced scales",
+			desc: [`§7Increases the stats of Shark Armor by §a${0.2*this.level}%`]
+		};
+	}
+
+	get third() {
+		return {
+			name: "§6Feeding frenzy",
+			desc: [`On kill gain §c${0.5 * this.level}${symbols.strength} Damage §7and §f${symbols.speed} speed for 5 seconds`]
```
Comment (metalcupcake5) [inline src/constants/petStats.js:1990]:
```suggestion
			desc: [`§7On kill gain §c${0.5 * this.level}${symbols.strength} Damage §7and §f${symbols.speed} Speed §7for 5 seconds`]
```

**Review by metalcupcake5 — CHANGES_REQUESTED** [review]
Also please move the multiplier above the return statement to keep with the other pet stat classes.
For example, turn this
```
get first() {
    return {
        name: "§6Blood Scent",
        desc: [`§7Deal up to §c+${round(0.25*this.level,1)}% ${symbols.strength} §7Damage based on the enemy's missing health`]
    };
}
```
into this
```
get first() {
    let mult = 0.25;
    return {
        name: "§6Blood Scent",
        desc: [`§7Deal up to §c+${round(this.level * mult,1)}% ${symbols.strength} §7Damage based on the enemy's missing health`]
    };
}
```

**Comment by metalcupcake5** [comment]
Also you can get the head from the NBT, should be pretty easy.

**Comment by felix642** [comment]
How am I suppose to get the NBT ?

**Comment by metalcupcake5** [comment]
Using sba you can copy item NBT

**Comment by felix642** [comment]
image

File: src/constants/petStats.js [hunk 2]
```diff
@@ -1971,23 +1971,26 @@ class Megalodon extends Pet{
...
+			desc: [`§7Increases the stats of Shark Armor by §a${mult*this.level}%`]
 		};
 	}
 
 	get third() {
+		let mult = 0.5;
 		return {
 			name: "§6Feeding frenzy",
-			desc: [`On kill gain §c${0.5 * this.level}${symbols.strength} Damage §7and §f${symbols.speed} speed for 5 seconds`]
+			desc: [`§7On kill gain §c${mult * this.level}${symbols.strength} Damage §7and §f${symbols.speed} speed §7for 5 seconds`]
```
Comment (boblovespi) [inline src/constants/petStats.js:1993]: Speed should be capitalized, no?

File: src/constants/petStats.js [hunk 3]
```diff
@@ -1971,23 +1971,26 @@ class Megalodon extends Pet{
 	}
 
 	get first() {
+		let mult = 0.25
```
Comment (boblovespi) [inline src/constants/petStats.js:1974]: missing a semicolon
```suggestion
		let mult = 0.25;
```

**Review by boblovespi — CHANGES_REQUESTED** [review]

**Review by metalcupcake5 — APPROVED** [review]

**Review by boblovespi — APPROVED** [review]

**Pull request merged**